docs/content/en/getting_started/running-in-production.md

```diff
@@ -79,7 +79,9 @@ You can execute the following command to see the configuration:
 `docker-compose exec celerybeat bash -c "celery -A dojo inspect stats"`
 and see what is in effect.
 
-###### Asynchronous Imports
+#### Asynchronous Import
+
+**Please note: Asynchronous Import is currently an experimental feature which is still in development.  Please exercise caution with this method as results may be inconsistent.**
```

Comment:
```suggestion
**Please note: Asynchronous Import is currently an experimental feature.  Please exercise caution with this method as results may be inconsistent.**
```
Removing the development bit since we are in a feature freeze